docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+bot:
+  build: .
+  volumes:
+    - .:/web
+  ports:
+    - "8800:80"
+  command: /bin/bash -c "cd /web/;
```

Comment:
It changes. And somebody will spend time to find why this is not starting. 
Install latest requirements is always good. 
While person can forget to run `docker-compose build`